src/forms/components/encounter/ohri-encounter-form.tsx

```diff
@@ -418,6 +425,17 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
         setFields(fields_temp);
       });
     }
+    if (field.sectionDependants) {
+      field.sectionDependants.forEach(dependant => {
+        for (let i = 0; i < form.pages.length; i++) {
+          const section = form.pages[i].sections.find((section, _sectionIndex) => section.label == dependant);
```

Comment:
@samuelmale I see you only use the for `index` in one place. Why don't you use a `foreach` instead?

Reply:
@ebambo the forEach doesn't natively support breaking out of the loop

Reply:
For this specific case, we want to break immediately we hit a match